packages/wallet/src/redux/state.ts

```diff
@@ -132,6 +132,7 @@ export interface FundingState {
 export interface ChannelFundingState {
```

Comment:
Should this now be split into a union type `DirectFundingState | IndirectFundingState | VirtualFundingState`? That would also clean up some of the checks required above on the existence of the `fundingChannel`

Reply:
Created #663 for this.